lib/matplotlib/axes/_axes.py

```diff
@@ -215,11 +215,12 @@ def legend(self, *args, **kwargs):
             line.set_label('Label via method')
             ax.legend()
 
-        Specific lines can be excluded from the automatic legend element
-        selection by defining a label starting with an underscore.
-        This is default for all artists, so calling `.Axes.legend` without
-        any arguments and without setting the labels manually will result in
-        no legend being drawn.
+        .. note::
+            Specific artists can be excluded from the automatic legend element
+            selection by defining a label starting with an underscore, "_".
+            This is default for all artists, so calling `.Axes.legend` without
```

Comment:
```suggestion
            `_nolegend` is the default label for all artists, so calling `.Axes.legend` without
```

Reply:
That the default is '_nolegend_' is an implementation detail and not something an end user should know or rely on. Please find a wording without this detail.

Reply:
Edited - does it work now? I just felt that the this was a slightly ambiguous reference.